src/controllers/user/franceconnect.ts

```diff
@@ -54,27 +58,32 @@ export async function getFranceConnectOidcCallbackController(
     }
     const { code } = await z.object({ code: z.string() }).parseAsync(req.query);
 
-    const { nonce, state } = await FranceConnectOidcSessionSchema.parseAsync(
-      req.session,
-    );
+    const { nonce, state, redirectTo } =
+      await FranceConnectOidcSessionSchema.parseAsync(req.session);
     const franceconnectUserInfo = await getFranceConnectUser({
       code,
       currentUrl: `${HOST}${FRANCECONNECT_CALLBACK_URL}${req.url.substring(req.path.length)}`,
       expectedNonce: nonce,
       expectedState: state,
     });
 
-    const { id: user_id } = getUserFromAuthenticatedSession(req);
+    const { id: userId } = getUserFromAuthenticatedSession(req);
 
     const updatedUser = await updateFranceConnectUserInfo(
-      user_id,
+      userId,
       franceconnectUserInfo,
     );
     updateUserInAuthenticatedSession(req, updatedUser);
 
-    return res.redirect(
-      "/personal-information?notification=personal_information_update_via_franceconnect_success",
-    );
+    const userOrganizations = await getOrganizationsByUserId(userId);
```

Comment:
issue: previous certifications should be invalidated only if the fc identity has been changed.